src/homepage/howTo/deleted-messages-vault.html

```diff
@@ -0,0 +1,307 @@
+                                <li>supports before or equals operator</li>
+                                <li>supports after or equals operator</li>
+                            </ul>
+                        </li>
+                        <li>recipients(List of string)
+                            <ul>
+                                <li>supports contains operator</li>
+                            </ul>
+                        </li>
+                        <li>sender(String)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                            </ul>
+                        </li>
+                        <li>has attachment(Boolean)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                            </ul>
+                        </li>
+                        <li>subject(String)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                                <li>supports equal ignore case matching operator</li>
+                                <li>supports contains matching operator</li>
+                                <li>supports contains ignore case matching operator (with US locale)</li>
+                            </ul>
+                        </li>
+                    </ul>
+
+                    <header class="major">
+                        <h2><b>How to setup James with Deleted Messages Vault</b></h2>
+                    </header>
+
+                    <p>
+                        In this section, we will guide you to setup James with Deleted Message Vault by following below steps:
+                    </p>
+                    <ul>
+                        <li>Enable Deleted Messages Vault</li>
+                        <li>Make James uses Deleted Messages Vault by configuring Pre Deletion Hooks</li>
+                        <li>Starting James with enabled Deleted Message Vault by docker compose</li>
+                    </ul>
+
+                    <header class="major">
+                        <h3><b>Enable Deleted Messages Vault</b></h3>
+                    </header>
+
+                    <p>
+                        To do this, you have to create a configuration file <b>deletedMessageVault.properties</b>, then put it into <b>conf</b> directory of James.
+                        There are two available properties you may want to configure:
+                    </p>
+                    <ul>
+                        <li>
+                            <b>urlPrefix</b>: represent for the prefix of namespace Deleted Messages Vault will use to store deleted messages.
+                        </li>
+                        <li>
+                            <b>retentionPeriod</b>: represent for the period deleted messages allowed to be stored in Deleted Messages Vault.
+                        </li>
+                    </ul>
```

Comment:
I saw that there is an example a bit more down in a later section, but wouldn't it be better to put an example here instead proper to this section? (opened question)